src/main/java/com/google/sps/servlets/ChangeControllerToServlet.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that changes the controller of a Session. */
+@WebServlet("/change-controller-to")
```

Comment:
You can keep this as change-controller (same for the class name). The "to" part was just for the javascript function because it's easier to read